webpack/scenes/ContentViews/Create/CreateContentViewFormComponents.js

```diff
@@ -0,0 +1,93 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+import React from 'react';
+import { Popover, Flex, FlexItem } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+import ContentViewIcon from '../components/ContentViewIcon';
+
+
+export const LabelComposite = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerNone' }}><ContentViewIcon composite /></FlexItem>
+    <FlexItem>Composite Content View</FlexItem>
+  </Flex>
+);
+
+export const LabelComponent = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerNone' }}><ContentViewIcon composite={false} /></FlexItem>
+    <FlexItem>Component Content View</FlexItem>
+  </Flex>
+);
+
+export const LabelDependencies = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerSm' }}>Solve Dependencies </FlexItem>
+    <FlexItem>
+      <Popover
+        bodyContent={
+            __('This will solve RPM and module stream dependencies on every publish of this content view. ' +
+            'Dependency solving significantly increases publish time (publishes can take over three times as long) ' +
+            'and filters will be ignored when adding packages to solve dependencies. ' +
+            'Also, certain scenarios involving errata may still cause dependency errors.')
+          }
+        aria-label="Solve-dependencies"
+        position="right"
+      >
```

Comment:
Seems like we should DRY this up with webpack/scenes/ContentViews/Details/ContentViewInfo.js, looking at the structure sharing just the text would make the most sense IMO